frontend/src/DashboardView/components/TeamList/TeamList.styles.js

```diff
@@ -1,7 +1,110 @@
 import styled from 'styled-components';
+import mediaQueryFor from '../../../_global_styles/responsive_querie';
+import backgroundGradient from '../../../_global_styles/background_gradient';
 
 export const Container = styled.div`
+  position:relative;
+  top:50px;
+	width: 80%;
 	display: flex;
 	flex-direction: column;
 	margin: 0 auto;
+  margin-left: 0 5% 0 5%;
+  color:white;
+  font-family: Comfortaa;
+  background: ${backgroundGradient}
+
+	/* background-color:purple; */
+	h3 {
+		color: #fff;
+		margin: 0 auto;
+	}
+	${mediaQueryFor.mdDevice`
+      
+      width:100%;
+    `}
+`;
+
+const Form = styled.div`
```

Comment:
I haven't seen any impact in this project yet, but be careful about naming components similar to html elements, like Form and form. It can potentially cause weirdness and hard to trace bugs.